src/styles/product-card.css

```diff
@@ -0,0 +1,146 @@
+html {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.product-card {
+  width: 200px;
+}
+
+.product-card__container {
+  width: 200px;
+  border: 1px solid var(--color-border-secondary);
+  border-radius: 5px;
+  background-color: var(--color-bg);
+  padding-bottom: 16px;
+}
+
+.product-card__product-image {
+  width: 160px;
+  height: 166px;
+  margin: 0 19px;
+  display: flex;
+}
+
+.product-card__product-card-image {
+  width: 160px;
+  height: 134px;
+  align-self: flex-end;
+}
+
+.product-card__product-title {
+  margin: 0 16px;
+  width: 168px;
+  height: 76px;
+  display: flex;
+}
+
+.product-card__description {
+  padding: 0;
+  margin: 0;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-txt-primary);
+  width: 166px;
+  height: 36px;
+  align-self: flex-end;
+}
+
+.product-card__product-code {
+  margin: 0 16px;
+  height: 18px;
+  display: flex;
+}
+
+.product-card__product-code-description {
+  padding: 0;
+  margin: 0;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  height: 14px;
+  align-self: flex-end;
+}
+
+.product-card__review-block {
+  display: flex;
+  justify-content: space-between;
+  height: 32px;
+  margin: 0 16px;
+}
+
+.product-card__info {
+  height: 16px;
+  align-self: flex-end;
+}
+
+.product-card__info-description {
+  margin: 0;
+  padding: 0;
+  font-size: 10px;
+  margin-top: 2px;
+  margin-right: 2px;
+}
+
+.product-card__price-block {
+  margin: 0 16px;
+}
+
+.product-card__title {
+  display: flex;
+  justify-content: space-between;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  height: 42px;
+}
+
+.product-card__price-title {
+  align-self: flex-end;
+}
+
+.product-card__price-quantity {
+  color: var(--color-txt-primary);
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  align-self: flex-end;
+  margin-right: 2px;
+}
+
+.product-card__buttom-block {
+  margin: 0 16px;
+  height: 56px;
+  display: flex;
+}
+
+.product-card__button-action {
+  background: var(--color-background-btn);
+  border: none;
```

Comment:
Border should be here, but it should be transparent initially, then you just change the color on hover